packages/harmony/src/components/internal/MenuItem.tsx

```diff
@@ -1,22 +1,28 @@
-import { forwardRef, MouseEventHandler, Ref, useCallback } from 'react'
+import {
+  forwardRef,
+  MouseEventHandler,
+  ReactNode,
+  Ref,
+  useCallback
+} from 'react'
 
 import { CSSObject } from '@emotion/react'
 
-import { ButtonProps } from 'components'
 import { BaseButton } from 'components/button/BaseButton/BaseButton'
+import { BaseButtonProps } from 'components/button/BaseButton/types'
 import { IconComponent } from 'components/icon'
 import { Text } from 'components/text/Text'
 import { useTheme } from 'foundations'
 
 export type MenuItemProps<Value extends string> = Omit<
-  ButtonProps,
+  BaseButtonProps,
   'variant' | 'onChange'
 > & {
   variant?: 'option' | 'button'
-  label?: string
+  label?: ReactNode
```

Comment:
Can you just do a quick pass to check this doesn't break selects, filter buttons, etc